sdcm/tombstone_gc_verification_thread.py

```diff
@@ -29,6 +28,8 @@ def __init__(self, db_cluster: [BaseScyllaCluster, BaseCluster], duration: int,
         self.termination_event = termination_event
         self._thread = threading.Thread(daemon=True, name=self.__class__.__name__, target=self.run)
         self.db_cluster: [BaseScyllaCluster, BaseCluster] = db_cluster
+        node = next(node for node in self.db_cluster.data_nodes if node.db_up())
```

Comment:
this is not needed, you can always assign when needed, since you check before usage anyway.

Reply:
ok, removed.

Reply:
on second thought, this might be wrong since it fails like:
```
  File "/home/yarongilor/git/scylla-cluster-tests/sdcm/tombstone_gc_verification_thread.py", line 99, in run
    self._sstable_utils.verify_a_live_normal_node_is_used()
  File "/home/yarongilor/git/scylla-cluster-tests/sdcm/utils/sstable/sstable_utils.py", line 168, in verify_a_live_normal_node_is_used
    self.db_node = next(node for node in self.db_cluster.data_nodes if node.db_up())
```